theme/templates/generic/includes/rating.html

```diff
@@ -1,11 +1,30 @@
 {% load mezzanine_tags rating_tags i18n %}
-<h3>{% trans "Ratings" %}</h3>
-<span id="rating-{{ rating_object.id }}">
+
+<th>+1 Votes:</th>
+<td>
+    <span id="rating-{{ rating_object.id }}">
+    {% if rating_sum > 0 %}
+        {% if you_rated %}
+            {% if rating_sum == 1 %}
+                <span>You +1 this</span>
+            {% elif rating_sum == 2 %}
+                <span>You and 1 other +1 this</span>
```

Comment:
Doesn't the dynamic rating_sum doesn't cover this case?

Reply:
I'm confused from my comment too!  I was just pointing out that I think you can remove this block as the else block will create the same message when rating_sum equals 2.

Reply:
... Now I see what case this is covering, others is plural in the else block below.  Nevermind, nothing to see here.